host-spec/c03-transition.tm

```diff
@@ -626,26 +613,26 @@
 
   <subsection|Importing and Validating Block><label|sect-block-validation><label|sect-block-submission>
 
-  Block validation is the process by which the client asserts that a block is
-  fit to be added to the blockchain. This means that the block is consistent
-  with the world state and transitions from the state of the system to a new
-  valid state.
+  Block validation is the process by which a node asserts that a block is fit
+  to be added to the blockchain. This means that the block is consistent with
+  the world state and transitions from the current state of the system to a
+  new valid state.
 
   \;
 
-  Blocks can be handed to the Polkadot Host both from the network stack for
-  example by means of Block response network message (see Section
-  <reference|sect-msg-block-request> ) and from the consensus engine. Both the
-  Runtime and the Polkadot Host need to work together to assure block
-  validity. A block is deemed valid if the block author had the authorship
-  right for the slot during which the slot was built as well as if the
-  transactions in the block constitute a valid transition of states. The
-  former criterion is validated by the Polkadot Host according to the block
-  production consensus protocol. The latter can be verified by the Polkadot
-  Host invoking <verbatim|Core_execute_block> entry into the Runtime as
-  defined in section <reference|sect-rte-core-execute-block> as a part of the
-  validation process. Any state changes created by this function on
-  successful execution are persisted.
+  New blocks can be received by the Polkadot Host via other peers (see
+  Section <reference|sect-msg-block-request>) or from the Host's own
+  consensus engine (see Section <reference|sect-block-production>). Both the
+  Runtime and the Polkadot Host then need to work together to assure block
+  validity. A block is deemed valid if the block author had authorship rights
+  for the slot in which the block was produce as well as if the transactions
+  in the block constitute a valid transition of states. The former criterion
+  is validated by the Polkadot Host according to the block production
+  consensus protocol. The latter can be verified by the Polkadot Host
+  invoking <verbatim|Core_execute_block> entry into the Runtime as defined in
+  section <reference|sect-rte-core-execute-block> as a part of the validation
+  process. Any state changes created by this function on successful execution
+  are persisted.
```

Comment:
> persisted

Basti hates this word :) Basically, executing a block simply creates a new state (state root), which the state machine can navigate to/from. I guess this is fine as it is. Let me know.

Reply:
Good to know. But as far as I can tell, "persisted" was also used here before.

Reply:
Yep, I think I originally wrote that.